auto-dark.el

```diff
@@ -148,19 +146,67 @@ Argument APPEARANCE should be light or dark."
   (when (timerp auto-dark--timer)
     (cancel-timer auto-dark--timer)))
 
+(defun auto-dark--register-dbus-listener ()
+  "Ask D-Bus to send us a signal on theme change and add a callback
```

Comment:
Hello again @J3RN! Reopening this review since I should've oriented you better (my bad) :)

Part of keeping a package on MELPA is following this rules:
- Should clean pass byte compiling
- Should be OK with `checkdock` 

<img width="450" alt="image" src="https://user-images.githubusercontent.com/16169950/210099311-3d6acb57-40c1-4dc4-ab46-51abc9d50588.png">

At this line, checkdoc complains the first line is not a complete sentence. It means that the first line should be a short sentence ending with period (.). Two or more lines are allowed, but the first one should be a complete short sentence.

Since both functions have the same comment, something to differ them would be better, like:
```
(defun auto-dark--register-dbus-listener ()
  "Register a listener function to D-Bus.
Ask D-Bus to send us a signal on theme change and add a callback
to change the theme."
```

and 

```
(defun auto-dark--unregister-dbus-listener ()
  "Unregister a listener function to D-Bus.
Ask D-Bus to send us a signal on theme change and add a callback
to change the theme."
```